src/models/clients.js

```diff
@@ -0,0 +1,31 @@
+const { Schema, model } = require('mongoose');
+
+const clientSchema = Schema({
+	name: {
+		type: String,
+	},
+	phone: {
+		type: String,
+	},
+	carBrand: {
+		type: String,
+	},
+	carModel: {
+		type: String,
+	},
+	licensePlate: {
+		type: String,
+	},
+	email: {
+		type: String,
+	},
+	favorite: {
+		type: Boolean,
+	},

```

Comment:
I don't think object with configuration ```{ versionKey: false, timestamps: true }```.
Is it okay?

Reply:
need to discuss

Reply:
Sorry, I don't see*

Reply:
I think it`s no need adding object with configuration to `Clients` model.

Reply:
I think we don't need versionsKey, but some moment we can need createdAt and updatedAt.